website/src/pages/documents/document.page.tsx

```diff
@@ -172,17 +215,31 @@ const TranslationTab = ({ doc }: { doc: Document }) => {
         {/*<PhoneticsPicker onSelect={setPhoneticRepresentation} />*/}
       </SolidSticky>
 
-      <article className={css.annotationContents}>
-        <DocumentContents
-          {...{
-            experienceLevel,
-            doc,
-            openDetails,
-            tagSet,
-            phoneticRepresentation,
-          }}
-        />
-      </article>
+      <section className={css.contentContainer}>
```

Comment:
I wouldn't use the `section` semantic element here because you are just grouping two things together, it's not really a thematic section of content. I would probably use a `div` here.